lib/team.rb

```diff
@@ -15,11 +15,11 @@ def initialize(team_params)
 
   def info
     {
-      team_id: @team_id,
-      franchise_id: @franchise_id,
-      team_name: @team_name,
-      abbreviation: @abbreviation,
-      link: @link
+      "team_id" => @team_id,
```

Comment:
thanks for addressing these!

Reply:
oh wait, I'm realizing this isn't referring to the headers - whoops :)
still good call on this method!